src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import classNames from 'classnames';
+import dropdownStyles from './styles';
+
+interface OptionItem {
+  id: number,
+  value: string
+}
+
+interface OptionsListProps {
+  options: OptionItem[],
+}
+
+interface DropdownProps {
+  name: string,
+  options: OptionItem[],
+  title: string,
+  className?: string,
+}
+
+const OptionsList: React.FC<OptionsListProps> = (props) => {
+  
+  const { options } = props;
+
+  if (options && options.length) {
+    return (
+      <div>
+        {options.map((opt) => (
+          <MenuItem key={opt.id} value={opt.id}>{opt.value}</MenuItem>
+        ))}
+      </div>
+    );
+  }
+
+  return null;
+}
+
+const Dropdown: React.FC<DropdownProps> = (props) => {
+
+  const { className, name, options, title } = props;
+  const [dropdownValue, setDropdownValue] = React.useState<string | number>();
+  const classes = dropdownStyles();
+  const rootClass: string = classNames(classes.root, className);
+
+  const handleChange = (event: React.ChangeEvent<{ value: any }>) => { 
```

Comment:
Just wondering if there's a more specific type that we give ChangeEvent< { value: any } >?

Reply:
normally you'll pass in the html class of the element triggering the event, eg. `React.ChangeEvent<HTMLInputElement>`

Reply:
I had to do that due to the interface definition of the select input props in Material UI: https://unpkg.com/@material-ui/core@4.0.0-alpha.8/Select/SelectInput.d.ts

onChange?: (
    event: React.ChangeEvent<{ name?: string; **value: unknown** }>,
    child: React.ReactNode,
  ) => void;

Here we have a discussion about that specific issue: https://github.com/mui-org/material-ui/issues/15400